Pyrado/scripts/training/qq-su_ppo_sprl.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
oh, so this is your sprl file, well (1) rename it to the comment above please, and (2) what do you need the other file for?